### DISCUSSION TITLE
Please avoid non-essential PRs at this time

### DISCUSSION THREAD

**Discussion by KirillOsenkov** [KirillOsenkov/MSBuildStructuredLog#816] [Announcements]
If you're here, it means you care more than 99% of people. Thank you!

I love and support open source and normally I want to encourage collaboration and PRs.

However. I'm extremely burned out. Maintenance on this repo has been incredibly taxing for me. For the foreseeable future I reserve the right to reject non-essential PRs as they direct my time and attention from the things I deem important.

There have been recent instances where well-meaning PR have caused a nightmare of support that I have to deal with. As an example, you can read this thread: https://github.com/KirillOsenkov/MSBuildStructuredLog/pull/801#issuecomment-2323676473. A seemingly helpful and innocuous PR has wasted my whole Sunday that I had to take away from my family and what I would want to do, and instead I feverishly worked to try and mitigate the regression. As a result, all users of the app are stuck on version 2.2.317 because auto-update doesn't work. Now I have to post announcements everywhere and notify people of how to update, and deal with a never-ending stream of support requests. 

In another recent instance, new unwanted dependencies have been introduced to the NuGet package inadvertently: https://github.com/KirillOsenkov/MSBuildStructuredLog/pull/798#discussion_r1742775769

Basically all this to say that I could either spend whatever remaining spare time I have with my family or review your PRs instead, and deal with breakages.

If there are tangible, immediate, obvious benefits to the users, by all means send a PR. Otherwise you can consider filing a bug, and if I mark it as up-for-grabs, then go ahead and submit a PR. If it's not critical, please avoid sending a PR at this time. Let me focus on what is important, and let me determine what the priorities of the project should be. My time and attention is an extremely limited and valuable resource.

Thank you for your understanding!
